packages/libs/deer/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+//! Intentionally left blank for now!
+
+#![cfg_attr(not(feature = "std"), no_std)]
+#![warn(
+    missing_docs,
+    unreachable_pub,
+    clippy::pedantic,
+    clippy::nursery,
+    clippy::undocumented_unsafe_blocks
+)]
+#![allow(clippy::redundant_pub_crate)]
+#![allow(clippy::module_name_repetitions)]
+
+extern crate alloc;
+extern crate core;
```

Comment:
`core` is imported by default (Either since the `2018` or `2021` edition)

Reply:
removed in [bb95c16](https://github.com/hashintel/hash/pull/1163/commits/bb95c166353807767082715d378f75bad27392b7)